oshi-core/src/main/java/oshi/software/os/mac/MacOperatingSystem.java

```diff
@@ -266,18 +270,19 @@ public List<OSService> getServices() {
         }
         // Get Directories for stopped services
         ArrayList<File> files = new ArrayList<>();
-        File dir = new File(SYSTEM_LIBRARY_LAUNCH_AGENTS);
-        if (dir.exists() && dir.isDirectory()) {
-            files.addAll(Arrays.asList(dir.listFiles((f, name) -> name.toLowerCase().endsWith(".plist"))));
-        } else {
+
+        try (Stream<Path> file = Files.list(Path.of(SYSTEM_LIBRARY_LAUNCH_AGENTS))) {
```

Comment:
as well as `Paths.of` since [11](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/nio/file/Path.html#of(java.lang.String,java.lang.String...))...

Reply:
[Same answer as above.](https://endoflife.date/java)